yarn-project/circuits.js/src/structs/avm/avm.ts

```diff
@@ -613,21 +616,14 @@ export class AvmCircuitInputs {
     return this.toBuffer().toString('hex');
   }
 
-  /**
-   * Is the struct empty?
-   * @returns whether all members are empty.
-   */
-  isEmpty(): boolean {
```

Comment:
Removing this as it was only used in `avm.test.ts`, and I didn't want to add `isEmpty` to `AvmCircuitPublicInputs` and all its child types.
In real world we might never need to check if the inputs to the AVM is empty or not.